source/client/hud.qc

```diff
@@ -1925,6 +1925,35 @@ void(float width, float height) HUD_RoundStopWatch =
 	Draw_String([width - (getTextWidth(stopwatch, 12)) - 2, 16], stopwatch, [12, 12], [1, 1, 1], 1, 0);
 }
 
+/*******************
+*     HUD UPS      *
+*******************/
+
+void() Draw_UPS =
```

Comment:
How about naming this to `HUD_PlayerDebugInfo`, and using our own cvar `scr_playerdebuginfo` instead? We can alias `show_speed` to it to keep it intuitive to QW folks, `1` will just display velocity in `qu/s` and `2` will also show origin and angles.

Reply:
addressed in [167e85a](https://github.com/nzp-team/quakec/pull/95/commits/167e85acf9cb0eac369006d70baff695129fed14)